backend/workflow_manager/workflow/workflow_helper.py

```diff
@@ -272,14 +275,20 @@ def run_workflow(
                 single_step=single_step,
                 hash_values_of_files=hash_values_of_files,
             )
-            log_id = str(execution_service.execution_log_id)
-            if log_id:
-                log_id = str(log_id)
+            # Update pipeline status
+            if updated_execution.status != ExecutionStatus.ERROR.value:
+                PipelineProcessor.update_pipeline(
+                    pipeline_id, Pipeline.PipelineStatus.SUCCESS
```

Comment:
I believe pipeline_id Nonetype handled inside PipelineProcessor.update_pipeline method, right?

Reply:
Yes it is, it can be refactored later if we wish to move the check outside